docs/UserGuide.md

```diff
@@ -370,14 +390,19 @@ please use [List Recipes](#436-list-recipes) command.
 
 **Additional Information:**<br />
 - Current requirements for a keyword:
-    1. Keyword is case-insensitive.
-       i. e.g. Finding with keyword: "mee" will match with "Maggie Mee"
-    2. Any keyword must match a full word in the name of the recipe.
-       i. e.g. Finding with keyword: "mee" will match with "Maggie Mee", "mee Goreng", etc. but not "meek"
-       ii. e.g. Finding with keywords: "salad Chicken burger" will match "Fried Chicken", "Burger Chicken", "Salad",
-       "Chicken Salad", etc. but not "chickenburger"
+    1. Keyword is case-insensitive. 
+        1. e.g. Finding with keyword: "mee" will match with "Maggie Mee"
```

Comment:
This example is showing 2 behaviours. Should only show mee matching with Mee for this part